day2_exercises/into4D.py

```diff
@@ -0,0 +1,113 @@
+#%% Into four dimensions
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Nov  7 13:46:24 2015
+Exercise for Fall 2015
+@author: Elissaios
+"""
+# Compatibility with Python 3
+from __future__ import print_function, division
+
+# Standard imports of libraries
+import numpy as np
+import nibabel as nib
+import matplotlib.pyplot as plt
+
+# Declarations
+TR = 2.5
+std_vol = []
+ave_task = []
+ave_rest = []
+outlier_task = []
+outlier_rest = []
+
+# Input data
+folder = r"C:\Users\Elissaios\Box Sync\Projects\fMRI\analysis1";
+filePath = folder + r"\ds114_sub009_t2r1.nii";
+condPath = folder + r"\ds114_sub009_t2r1_cond.txt";
+img = nib.load(filePath)
+data = img.get_data()
+#print(img.shape)
+task = np.loadtxt(condPath)
+task[:,0:2] = task[:,0:2]/TR  # change sec to TR for first two columns (2 is for 1+1)
```

Comment:
You might hit a problem here, because `task` may be an array of integers, in which case this line will get truncated to integers.  Better to do something like:

```
ons_dur = task[:,0:2] / TR
```